### DISCUSSION TITLE
What is Flash Player ؟

### DISCUSSION THREAD

**Discussion by MM25Zamanian** [AliMD/alwatr#69] [Q&A] [answered]
What is Flash Player and in what layer of the browser was it used in the past?

> فلش پلیر چیست و در گذشته در چه لایه ای از مرورگر استفاده می شد؟

**Comment by alimd** [comment, marked as answer]
مثل یک پلاگین بود که پسوند خودش رو با دسترسی کامل به سیستم اجرا میکرد
همونطور که الان جاوا پلت های سنتی در مرورگر اجرا میشه
